config/secrets.yml

```diff
@@ -13,13 +13,28 @@
 development:
   secret_key_base: 139e0d078b31c72aeb24893f994138f76fc3dc8f60d6964cc5cca0383b0cd54872eb3ad15f36d96232bc02b8ccc247d9ff0e63f969d3dc147b233591ce4e8c01
   link_checker_api_secret_token: h830foainflkwn439qfhla
+  GOOGLE_CLIENT_EMAIL: 'linkstatusuploader@badlinksexporter.iam.gserviceaccount.com'
+  GOOGLE_PRIVATE_KEY: 'abcdefg'
+  GOOGLE_EXPORT_ACCOUNT_ID: '12345'
+  GOOGLE_EXPORT_CUSTOM_DATA_IMPORT_SOURCE_ID: 'UA-XXXXX-X'
+  GOOGLE_EXPORT_TRACKER_ID: 'abcdefg'
 
 test:
   secret_key_base: d3e0f487c481073197624c84467c2278040043c05a721cdb344e627e37889c87979165ed276c87fa5ed2336dbc164bfb155f5995aad04dd4434ec304fda911a8
   link_checker_api_secret_token: fshur3h89fhu4jh3iouf90
+  GOOGLE_CLIENT_EMAIL: 'linkstatusuploader@badlinksexporter.iam.gserviceaccount.com'
+  GOOGLE_PRIVATE_KEY: 'abcdefg'
+  GOOGLE_EXPORT_ACCOUNT_ID: '12345'
+  GOOGLE_EXPORT_CUSTOM_DATA_IMPORT_SOURCE_ID: 'UA-XXXXX-X'
+  GOOGLE_EXPORT_TRACKER_ID: 'abcdefg'
 
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
 production:
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
   link_checker_api_secret_token: <%= ENV["LINK_CHECKER_API_SECRET_TOKEN"] %>
+  GOOGLE_CLIENT_EMAIL: 'linkstatusuploader@badlinksexporter.iam.gserviceaccount.com'
+  GOOGLE_PRIVATE_KEY: 'abcdefg'
+  GOOGLE_EXPORT_ACCOUNT_ID: '12345'
+  GOOGLE_EXPORT_CUSTOM_DATA_IMPORT_SOURCE_ID: 'UA-XXXXX-X'
+  GOOGLE_EXPORT_TRACKER_ID: 'abcdefg'
```

Comment:
Don't think we want these entries in secrets.yml now we're saying we will use env vars.